ni_measurementlink_service/measurement/service.py

```diff
@@ -3,7 +3,9 @@
 from __future__ import annotations
 
 from threading import Lock
-from typing import Any, Callable, Dict, TypeVar
+from typing import Any, Callable, Dict, TypeVar, List
```

Comment:
[O.1.4] ✔️ DO List imports in alphabetical order 💻

https://github.com/ni/python-styleguide/blob/main/docs/Coding-Conventions.md